packages/eslint-plugin/docs/rules/i18n-hyphenated-range.md

```diff
@@ -0,0 +1,23 @@
+# Disallow using a hyphenated numerical ranges in translatable strings (i18n-hyphen-range)
```

Comment:
Nitpick: Update `i18n-hyphen-range` to use the actual rule's name `i18n-hyphenated-range`.

```suggestion
# Disallow using a hyphenated numerical ranges in translatable strings (i18n-hyphenated-range)
```